<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgSimpleFilter_blob</class>
 <widget class="QDialog" name="CQDlgSimpleFilter_blob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blob Detection</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>321</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqThreshold">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqDiffColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Use a different color for each blob</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>API return values</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>(1) blob count, (2) value count per blob = vCnt, (3) blob1 size,
(4) blob1 orientation, (5) blob1 position x, (6) blob1 position y,
(7) blob1 width, (8) blob1 height, ..., (3+vCnt+0) blob2 size,
(3+vCnt+1) blob2 orientation, etc.</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="qqMinSize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ignore blobs smaller than</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqThreshold</tabstop>
  <tabstop>qqDiffColor</tabstop>
  <tabstop>qqOkCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
